### PR TITLE
Rendering plugins with Jinja2.

### PR DESCRIPTION
**Pull request by l9kd1**
Related to https://github.com/UCL-INGI/ICTV/pull/29

### CODE REVIEW

File: ictv/plugins/editor/templates/editor.html [hunk 1]
```diff
@@ -475,31 +488,32 @@ <h4 class="modal-title">Your slide looks a bit overweight.</h4>
...
                     } else {
-                        $$element.removeClass('hover');
+                        $element.removeClass('hover');
                     }
                 });
                 render();
                 render();
             } else {
-                window.location = window.location;
+                console.log("wtf");
```
Comment (anthonygego) [inline ictv/plugins/editor/templates/editor.html:515]: :eyes:

Comment (l9kd1) [reply ictv/plugins/editor/templates/editor.html]: Woops, this is solved now. 
With my apologies.

**Review by Drumor — COMMENTED** [review]
Looks globally good but didn't test yet.

File: ictv/plugins/rss/app.py [hunk 2]
```diff
@@ -50,17 +52,19 @@
...
-    app.renderer = web.template.render(os.path.join(os.path.dirname(__file__), 'templates'),
-                                       base=os.path.join(get_root_path(), 'templates', 'base'),
-                                       globals={'session': ictv_app.session,
-                                                'get_feedbacks': get_feedbacks,
-                                                'get_next_feedbacks': get_next_feedbacks,
-                                                'pop_previous_form': pop_previous_form,
-                                                'UserPermissions': UserPermissions,
-                                                'str': str, 'sidebar_collapse': True, 'show_header': False,
-                                                'show_footer': False, 'User': User})
+    print(ictv_app.session.__dict__)
```
Comment (anthonygego) [inline ictv/plugins/rss/app.py:55]: I guess this can be removed :D

**Comment by anthonygego** [comment]
LGTM.

**Pull request merged**